Factory for entities can now produce failed accessions

ttung opened the pull request:
### Description
`sequencing_read_factory` can now produce failed accessions.

### Test plan
pytest

phoenixAja reviewed (approved):
cool, do you need to update schema.ddl too or does this not really fit in there?

ttung commented:
No, this isn't part of the schema.

Commit:
Factory for entities can now produce failed accessions

### Description
`sequencing_read_factory` can now produce failed accessions.

### Test plan
pytest
